package.json

```diff
@@ -11,16 +11,24 @@
     "yarn": "^1.12.3"
   },
   "scripts": {
-    "dev": "webpack-dev-server --hot --inline  --config webpack.dev.js",
-    "build": "webpack --config webpack.prod.js",
-    "lint": "standard --fix"
+    "dev": "webpack-dev-server --hot --inline --config webpack.dev.ts",
+    "build": "webpack --config webpack.prod.ts",
+    "lint": "standard *.ts --fix"
```

Comment:
Replace this command with `tslint` (standard is no longer available in this project)